store/blocks.json

```diff
@@ -0,0 +1,5 @@
+{
+    "checkout-summary": {
+        "blocks": ["coupon"]
+    }
+}
```

Comment:
The spacing shouldn't be 2 instead of 4?

Reply:
Changed it!